spec.bs

```diff
@@ -1141,60 +1145,83 @@ element with arbitrary URLs *not* produced by config-generating APIs for develop
-     [=size/height=].
+<div algorithm="contentHeight getter">
+  The {{FencedFrameConfig/contentHeight}} IDL attribute getter steps are to return [=this=]'s
+  [=fencedframeconfig/contentHeight=].
 </div>
 
-<div algorithm="contentWidth getter">
-  The {{FencedFrameConfig/contentWidth}} IDL attribute getter steps are:
+<div algorithm>
+  The <dfn method for=FencedFrameConfig>setSharedStorageContext(|contextString|)</dfn> method steps
+  are to set [=this=]'s [=fencedframeconfig/sharedStorageContext=] to |contextString|.
+</div>
 
-  1. If [=this=]'s [=fencedframeconfig/config=]'s [=fenced frame config/content size=] is null,
-     return null.
+<div algorithm="FencedFrameConfig serializer">
+  {{FencedFrameConfig}} objects are [=serializable objects=]. Their [=serialization steps=], given
+  |value|, |serialized|, and |forStorage| are:
 
-  1. If [=this=]'s [=fencedframeconfig/config=]'s [=fenced frame config/content size=]'s [=content
-     size/visibility=] is "<a for=visibility>`transparent`</a>", return the [=fenced frame
-     config/content size=]'s [=content size/value=]'s [=size/width=].
+  1. If |forStorage| is true, then throw a {{DataCloneError}} {{DOMException}}.
+  
+  1. Set |serialized|.`[[Urn]]` to the value of |value|'s [=fencedframeconfig/urn=] attribute.
 
-  1. Otherwise, return the `"opaque"` {{OpaqueProperty}}.
-</div>
+  1. Set |serialized|.`[[SharedStorageContext]]` to the value of |value|'s [=fencedframeconfig/
+     sharedStorageContext=] attribute.
 
-<div algorithm="contentHeight getter">
-  The {{FencedFrameConfig/contentHeight}} IDL attribute getter steps are:
+  1. Set |serialized|.`[[ContainerWidth]]` to the value of |value|'s [=fencedframeconfig/
+     containerWidth=] attribute.
 
-  1. If [=this=]'s [=fencedframeconfig/config=]'s [=fenced frame config/content size=] is null,
-     return null.
+  1. Set |serialized|.`[[ContainerHeight]]` to the value of |value|'s [=fencedframeconfig/
+     containerHeight=] attribute.
+
+  1. Set |serialized|.`[[ContentWidth]]` to the value of |value|'s [=fencedframeconfig/
+     contentWidth=] attribute.
+
+  1. Set |serialized|.`[[ContentHeight]]` to the value of |value|'s [=fencedframeconfig/
+     contentHeight=] attribute.
 
-  1. If [=this=]'s [=fencedframeconfig/config=]'s [=fenced frame config/content size=]'s [=content
-     size/visibility=] is "<a for=visibility>`transparent`</a>", return the [=fenced frame
-     config/content size=]'s [=content size/value=]'s [=size/height=].
 
-  1. Otherwise, return the `"opaque"` {{OpaqueProperty}}.
 </div>
 
-<div algorithm>
-  The <dfn method for=FencedFrameConfig>setSharedStorageContext(|contextString|)</dfn> method steps
-  are:
+<div algorithm="FencedFrameConfig deserializer">
+  Their
+  [=deserialization steps=], given |serialized|, |value|, and <var ignore> targetRealm</var> are:
+
+  1. Initialize |value|'s [=fencedframeconfig/urn=] attribute to |serialized|.`[[Urn]]`.
```

Comment:
I don't think there should be wrapped in backticks. Just normal text (escaping the brackets if needed?).

Reply:
They need to be escaped, but I can get away with just escaping the first bracket.